scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```
@@ -32,7 +32,7 @@
 - name: Collect list of installed packages repo of origin from zypper
   shell: |
     if [ {{ hostvars[item_virtual_hosts].ansible_os_family }} = "Suse" ];then
-      zypper --xmlout se -s -i -t package
+      ZYPP_LOCKFILE_ROOT=/root; zypper --xmlout se -s -i -t package
```

You shouldn't need the `;`.

```suggestion
      ZYPP_LOCKFILE_ROOT=/root zypper --xmlout se -s -i -t package
```

---

@nicolasbock Yeah it is possible. I just wanted to make it clear that the command itself is `zypper`. But I can remove it if it is a problem.

---

I am not following @djz88 :smile: With the `;` those are two separate commands. But you are not exporting `ZYPP_LOCKFILE_ROOT` and the `zypper` command will not see the value you set it to. You need to either export that variable if you want to leave the `;` or remove the `;` to implicitly export the variable to the `zypper` command.

---

`;` is just  a command delimiter,right? All it runs in the same session in the same shell. So it is saved in the environment variable and therefore zypper can read it. Or am I wrong?